internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -105,25 +105,25 @@ func (ksd KubernetesServiceDeployer) SetUp(ctxt ServiceContext) (DeployedService
 	// to deploy Agent Pod. Because of this, hostname inside pod will be equal to the name of the k8s host.
 	ctxt.Agent.Host.NamePrefix = "kind-control-plane"
 	return &kubernetesDeployedService{
-		ctxt:           ctxt,
-		definitionsDir: ksd.definitionsDir,
+		ctxt:            ctxt,
+		definitionsPath: ksd.definitionsPath,
 	}, nil
 }
 
 func (ksd KubernetesServiceDeployer) installCustomDefinitions() error {
-	logger.Debugf("install custom Kubernetes definitions (directory: %s)", ksd.definitionsDir)
+	logger.Debugf("install custom Kubernetes definitions (directory: %s)", ksd.definitionsPath)
 
-	definitionPath, err := findKubernetesDefinitions(ksd.definitionsDir)
+	definitionsPath, err := checkKubernetesDefinition(ksd.definitionsPath)
```

Comment:
I'll be inclined to only return the error and use the original variable like suggested above.